src/rest-api-link.js

```diff
@@ -10,9 +10,18 @@ const getTypeNameFromDirective = directive => {
   return typeArgument.value.value;
 }
 
+const getRouteFromDirective = directive => {
```

Comment:
I think apollo-utilities has a method for getting directives from the AST, but I don't think we need to worry about it yet. There will be additions to the utilities package from this project I'm sure of so lets copy paste / import where it makes sense to be able to move faster. We can always remove code later 👍